examples/angularjs-perf/index.html

```diff
@@ -8,45 +8,51 @@
 		<style>[ng-cloak] { display: none; }</style>
 	</head>
 	<body>
-		<section id="todoapp" ng-controller="TodoCtrl">
+		<section id="todoapp" ng-controller="TodoCtrl as TC">
```

Comment:
`TC` in allcaps is unusual. it's a variable name, so it should be camelcase or all lowercase if just the first letters.